core/arch/arm/mm/core_mmu.c

```diff
@@ -101,6 +101,10 @@ static struct memaccess_area nsec_shared[] = {
 	MEMACCESS_AREA(CFG_SHMEM_START, CFG_SHMEM_SIZE),
 };
 
+#ifdef CFG_TEE_SDP_MEM_BASE
```

Comment:
Ok to condition SDP support to `CFG_WITH_SDP`.

Here, with or without (sic?) SDP support, if a platform need to register only 1 SDP memory, it can set generic `CFG_TEE_SDP_MEM_BASE/_SIZE` with the right values.

I think platform really willing to implement SDP, will simply add one or more `register_sdp_mem(...);` in their platform **main.c**.

`CFG_TEE_SDP_MEM_BASE/_SIZE` is only here to help testing wihtout hacking into the platform code.